src/Core_Command.php

```diff
@@ -108,6 +108,9 @@ function check_update( $_, $assoc_args ) {
 	 * [--force]
 	 * : Overwrites existing files, if present.
 	 *
+	 * [--skip-content]
+	 * : Download the latest version of WP without the default themes and plugins
```

Comment:
Can we include some mention that this only works when locale is `en_US`?

Reply:
For sure, I've just updated.